docs/guide/built-in-rules-composite.md

```diff
@@ -39,15 +39,15 @@ final class UsernameRuleSet extends Composite
 }
 ```
 
-And use it the same as one regular rule:
+And use it just like a single regular rule:
```

Comment:
Previous variant was confusing since "one" is a replacement word: https://lingbase.com/en/english/grammar/one-ones